wp-content/themes/pub/wporg-learn-2024/patterns/site-breadcrumbs.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Title: Site Breadcrumbs
+ * Slug: wporg-learn-2024/site-breadcrumbs
+ * Inserter: no
+ */
+
+// Ensure breadcrumbs are displayed only when there are at least 3 levels.
+$block_content = '<!-- wp:wporg/site-breadcrumbs {"fontSize":"small"} /-->';
+$parsed_content = do_blocks( $block_content );
+preg_match_all( '/<span\b[^>]*>/', $parsed_content, $matches );
+$breadcrumb_level = count( $matches[0] );
+$show_breadcrumbs = $breadcrumb_level >= 3;
+
+?>
+
+<?php if ( $show_breadcrumbs ) : ?>
+	<!-- wp:group {"className":"wporg-breadcrumbs","align":"full","style":{"spacing":{"padding":{"top":"23px","bottom":"23px","left":"var:preset|spacing|edge-space","right":"var:preset|spacing|edge-space"}}},"backgroundColor":"white","layout":{"type":"flex","flexWrap":"wrap","justifyContent":"space-between"}} -->
+	<div class="wporg-breadcrumbs wp-block-group alignfull has-white-background-color has-background" style="padding-top:23px;padding-right:var(--wp--preset--spacing--edge-space);padding-bottom:23px;padding-left:var(--wp--preset--spacing--edge-space)">
```

Comment:
I think this should be 18px top/bottom padding, to match the breadcrumbs on the other sites.

Reply:
Thanks. @fcoveram Could

Reply:
Thanks 👍  @fcoveram, do you mind also updating styles in the figma? The top/bottom padding for the breadcrumbs:
<img width="1255" alt="image" src="https://github.com/WordPress/Learn/assets/18050944/5223d68d-23b2-40c6-a88d-a95a8b352b29">

Reply:
Sure. I will apply the change